plugins/node/src/tasks/node.ts

```diff
@@ -38,13 +38,20 @@ export default class Node extends Task<typeof NodeSchema> {
       throw error
     }
 
+    const { execArgv } = process
+    // disable native fetch if supported by runtime
+    if (process.allowedNodeEnvironmentFlags.has('--no-experimental-fetch')) {
+      execArgv.push('--no-experimental-fetch')
```

Comment:
**question**: there might be side effects of mutating `process.execArgv`, should we clone the array?

Reply:
aaah yes good point!! lovely javascript